mysql-test/suite/rpl/r/rpl_bug85158.result

```diff
@@ -0,0 +1,60 @@
+include/rpl_init.inc [topology=1->2,2->3,1->3]
```

Comment:
Please drop the rpl_ prefix from the testcase name (just bug85158.result/test/cnf)

Reply:
Done.